packages/babel-preset-udemy-website/raw.js

```diff
@@ -37,7 +37,7 @@ module.exports = {
         // Legacy decorator transform with the old decorator behavior from Babel@5
         // There are some caveats as to how to set this up:
         // https://new.babeljs.io/docs/en/next/babel-plugin-proposal-decorators.html
-        ['@babel/plugin-proposal-decorators', { legacy: true }],
+        ['@babel/plugin-proposal-decorators', { legacy: true, decoratorsBeforeExport: true }],
```

Comment:
This is the configuration we need to make use of decorators the way we use them. In another PR, I will probably update our code to get rid of decoratorsBeforeExport.